ts/Data/DataPoolOptions.ts

```diff
@@ -28,13 +31,46 @@ import type { DataConnectorTypes } from './Connectors/DataConnectorType.js';
  * */
 
 export interface DataPoolOptions {
+    /**
+     * Link list to connector options + mention DataConnector.types
+     *
```

Comment:
This was text for your eyes only and should be replaced. :)

Reply:
🙈